doc/article/en-US/templating-custom-elements.md

```diff
@@ -272,15 +269,289 @@ Aurelia custom elements utilize the "slot based" content projection standard fro
+  </source-code>
+</code-listing>
+
+That was fun! But we can go deeper. What about slots, that target other slots with fallback content that generates slots...
+
+<code-listing heading="mixed-slot.html">
+  <source-code lang="HTML">
+    <template>
+      <div>
+        The first slot:
+        <div>
+          <slot name="slot1">
+            Default content for Slot 1
+          </slot>
+        </div>
+        The default slot:
+        <div>
+          <slot>
+            Default Content for the Default Slot
+          </slot>
+        </div>
+        The second slot:
+        <div>
+          <slot name="slot2">
+            The first fallback slot:
+            <div>
+              <slot name="fallbackSlot1">
+                Default Content for Fallback Slot 1
+              </slot>
+            </div>
+            The second fallback slot:
+            <div>
+              <slot name="fallbackSlot2"></slot>
+            </div>
+          </slot>
+        </div>
+      </div>
+    </template>
+  </source-code>
+</code-listing>
+
+<code-listing heading="slot-to-mixed-slot.html">
+  <source-code lang="HTML">
+    <template>
+      <require from="./mixed-slot"></require>
+
+      <div>
+        <mixed-slot>
+          <slot name="slot1" slot="slot1">Fallback Content for Projected Slot 1</slot>
+          <slot name="slot2" slot="fallbackSlot2">Fallback Content for Projected Slot 2</slot>
+        </mixed-slot>
+      </div>
+    </template>
+  </source-code>
+</code-listing>
+
+<code-listing heading="Usage">
+  <source-code lang="HTML">
+    <slot-to-mixed-slot>
+      <div slot="slot2">This is user content for slot 2. (should appear in fallbackSlot2)</div>
+    </slot-to-mixed-slot>
+  </source-code>
+</code-listing>
+
+<code-listing heading="The Composed Visual Tree">
+  <source-code lang="HTML">
+    <slot-to-mixed-slot>
+      <div>
+        The first slot:
+        <div>
+          Fallback Content for Projected Slot 1
+        </div>
+        The default slot:
+        <div>
+          Default Content for the Default Slot
+        </div>
+        The second slot:
+        <div>
+          <slot name="slot2">
+            The first fallback slot:
+            <div>
+              Default Content for Fallback Slot 1
+            </div>
+            The second fallback slot:
+            <div>
+              <div slot="slot2">This is user content for slot 2. (appearing in fallbackSlot2)</div>
+            </div>
+          </slot>
+        </div>
+      </div>
+    </slot-to-mixed-slot>
+  </source-code>
+</code-listing>
+
+## [Slot Implementation Limitations](aurelia-doc://section/9/version/1.0.0)
+
+At the time this document was written, no browser supports Shadow DOM v1 slots. Thus, Aurelia has implemented the specification. It also works with *template controllers* such as `if` and `repeat` which can dynamically generate content. We've fixed up our `@child` and `@children` decorators to understand the new model as well.
```

Comment:
@AshleyGrant
> It also works with *template controllers* such as `if` and `repeat` which can dynamically generate content

I think it worth rephrasing, so it would be very clear, that slots must not be nested inside template controllers, but slot content can have template controllers?

Note, I've added [tests for it to aurelia-template-lint](https://github.com/MeirionHughes/aurelia-template-lint/pull/27/files#diff-fa0e0cc88c52bc15a5a1ff71151bf248R47) to check the violation of this limitation at build-time ;)